test/index.js

```diff
@@ -296,16 +296,15 @@ describe('.dest()', function() {
 
   it('outputs files where the base ends in a separator', function(done) {
```

Comment:
I wasn't completely sure what this test does :-)
https://github.com/gulpjs/vinyl-prepare/pull/7
But I did notice the inversion, so now this tests (I think) that it return files with slash-less base, even when the input ends in separator.